packages/offix-client/src/apollo/conflicts/CompositeConflictListener.ts

```diff
@@ -0,0 +1,38 @@
+import { ConflictListener, ConflictResolutionData } from "offix-conflicts-client";
+
+
+/**
+ * Composite Conflict Listener class that can accept register and remove individual listener functions as needed
+ * Gets passed down to the conflict link
+ */
+export class CompositeConflictListener implements ConflictListener {
```

Comment:
Awesome, does this mean the changes aren't even breaking changes?

Reply:
exactly, you can still pass your original `conflictListener` in options if desired but it'll get registered into this Composite one.